spec/collections_publisher/creating_spec.rb

```diff
@@ -0,0 +1,40 @@
+feature "Publishing a topic on Collections Publisher", collections_publisher: true do
```

Comment:
This looks good, think the filename might need tweaking slightly to reflect the test